### PR TITLE
72 Add appointment status

### LINKED ISSUE #72: Add appointment status
As a user my appointments have a status

- Add a `status` to the appointment model
- That status will be an `enum`

### PR DESCRIPTION
**Pull request by lrs8810**
## Type of change
- [x] :hatching_chick: New feature (non-breaking change which adds functionality)
- [x] :page_with_curl: This change requires a documentation update

## Summary of changes
- Update documentation to reflect new response to get all appointments endpoint
- Add utils file with enum class 
- Add enum to Appointment model
- Update all appointments test to test status, notary name/id, and appointee name/id
- Update endpoint to format appointments
- Add format_appointment helper to format json response

## How Has This Been Tested?
- [x] :black_square_button: Integration testing

## Checklist:
- [x] :white_check_mark: I have performed a self-review of my own code
- [x] :page_facing_up: I have made corresponding changes to the documentation
- [x] :no_entry_sign: My changes generate no new warnings
- [x] :heavy_check_mark: I have added tests that prove my fix is effective or that my feature works
- [x] :100: New and existing unit tests pass locally with my changes

### CODE REVIEW

File: notary_now_api/helpers/format_appointment.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+            },
+        "appointee": {
+                "name": appointment.related_appointee.first_name + " " + appointment.related_appointee.last_name,
+                "id": appointment.appointee_id,
+            },
+        "status": appointment.get_appointment_result,
+        "time": appointment.time,
+        "date": appointment.date,
+        "location": appointment.location,
+    }
```
Comment (davidlopez) [inline notary_now_api/helpers/format_appointment.py:16]: Great work!

File: notary_now_api/migrations/0012_auto_20200222_2120.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        ('notary_now_api', '0011_appointment_status'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='appointment',
+            name='status',
+            field=models.IntegerField(choices=[(1, 'PENDING'), (2, 'COMPLETED'), (3, 'CANCELLED')], default=notary_now_api.utils.AppointmentStatuses['PENDING']),
+        ),
+    ]
```
Comment (davidlopez) [inline notary_now_api/migrations/0012_auto_20200222_2120.py:19]: New Migrations ✅

**Review by davidlopez — APPROVED** [review]

File: notary_now_api/helpers/format_appointment.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+def format_appointment(appointment):
```
Comment (BabsLabs) [inline notary_now_api/helpers/format_appointment.py:1]: Great use of a helper method to keep our code clean.

**Review by BabsLabs — APPROVED** [review]
That looks great. I like that we figured out enums and how to properly test them!

**Pull request merged**